### ISSUE TITLE
Boolean properties of EmailMessage that can be nullable, should be nullable by default

### ISSUE THREAD

**Issue by mikeduminy** [shawnmclean/Mandrill-dotnet#84]
The following snippet from the API docs (https://mandrillapp.com/api/docs/messages.JSON.html#method-send) shows how nullable boolean fields can be used, or not used.

``` javascript
{
  ... 
  "important": false,
  "track_opens": null,
  "track_clicks": null,
  "auto_text": null,
  "auto_html": null,
  "inline_css": null,
  "url_strip_qs": null,
  "preserve_recipients": null,
  "view_content_link": null,
  ...
}
```

This is crucial for things like stripping out query string parameters for click tracking aggregation. The property `url_strip_qs` on an message-level overrides the sending defaults defined within Mandrill.

Imagine your sending default was `true`. By forgetting to explicitly set this property to `true` or anything in .NET you unintentionally set it to `false`(as the field on the model is a boolean with a default of `false`), meaning you have to be sure to explicitly set these values in code, which defeats the purpose of global sending defaults - well, those that differ from the .Net default values.

TL;DR: The boolean fields in EmailMessage (and perhaps other models) should be nullable.
